tests/integration/test_charm_fork_repo.py

```diff
@@ -188,10 +188,7 @@ async def test_dispatch_workflow_failure(
             in logs
         ):
             assert run.jobs()[0].conclusion == "failure"
-            assert (
-                "Stopping execution of jobs due to repository setup is not compliant with policies"
-                in logs
-            )
```

Comment:
I think this message could be valuable and we could still check it.

To make the test goals more explicit, we could also change the docstring (L159) from
`assert: The workflow that was dispatched failed.` to `assert: The workflow that was dispatched failed and the reason is logged.`

Reply:
Updated the code